runtime/isp_run_app.py

```diff
@@ -39,6 +39,13 @@ def main():
     parser.add_argument("-p", "--policy", type=str, default="none", help='''
     Name of the installed policy to run or directory containing policy. Default is none
     ''')
+    parser.add_argument("-P", "--policy-full", type=str, help='''
```

Comment:
This short circuits an assumption in the runtime that the policy name is `osv.<runtime>.main.*`. In the `policy_tests` Makefile, we do this using an option `module` parameter as a prefix to the policy name. Would we prefer a similar option here?

Reply:
This option seems redundant. `-p` allows you to specify a local directory as a policy

Reply:
Without this option, `isp_run_app` calls this fn: https://github.com/draperlaboratory/hope-tools/blob/master/runtime/isp_utils.py#L44 which forces your policy name to an `isp.xxx.main.xxx` format (here https://github.com/draperlaboratory/hope-tools/blob/master/runtime/isp_run_app.py#L118)

Again using webapp as an example, we want the policy names to be `webapp.ppac` and `webapp.password` as they are application specific, so `osv.frtos.main.ppac` is a name that doesn’t really make sense.

Reply:
There may be a more principled name to handle this, like I suggested with a `module` argument, but this feature is not redundant.

Reply:
https://github.com/draperlaboratory/hope-tools/blob/master/runtime/isp_run_app.py#L120 see this line. If the policy is a directory, this naming convention is ignored.